src/diffcalc_API/errors/constraints.py

```diff
@@ -9,18 +9,16 @@
 
 
 class Codes(ErrorCodes):
-    CHECK_CONSTRAINT_EXISTS = 400
+    INVALID_CONSTRAINT_ERROR = 400
```

Comment:
I would be tempted to rename this something along the lines of `ErrorCodes` or `ConstraintErrorCodes` and rename `INVALID_CONSTRAINT_ERROR` to `INVALID_CONSTRAINT`

The current base class `ErrorCodes` could be renamed to `ErrorCodesBase` or be made into a mixin